path_rotate.go

```diff
@@ -70,7 +70,12 @@ func (b *backend) pathRotateCredentials() []*framework.Path {
 }
 
 func (b *backend) pathRotateRootCredentialsUpdate(ctx context.Context, req *logical.Request, _ *framework.FieldData) (*logical.Response, error) {
-	return nil, b.rotateRootCredential(ctx, req)
+	err := b.rotateRootCredential(ctx, req)
+	if err != nil {
+		b.Logger().Error("failed to rotate root credential:", "rotationID", req.Path, "err", err.Error())
```

Comment:
Should we be consistent and either use `err` or `error`?

Reply:
a bugbear across all these changes has been that the convention appears to be different between plugins; in this plugin specifically, `error` is used pretty consistently, but that's not the case for other plugins.

So we have a choice of either standardizing _this_ genus of log line across plugins, leaving the plugin itself inconsistent, or
standardizing within the plugin, leaving the rotation logging class of line inconsistent between plugins.